subprojects/ce-login/cli/CliCreateHsf.cpp

```diff
@@ -273,7 +334,7 @@ CeLogin::CeLoginRc cli::createHsf(int argc, char** argv)
     {
         createPrintHelp(argc, argv);
     }
-    else if (createValidateArgs(sArgs))
+    else if (true || createValidateArgs(sArgs))
```

Comment:
Please check this.  What is the intention?  Then please change the code (remove "true" or remove "createValidateArgs(sArgs)"?).

Reply:
I think this is just debug code that didn't get cleaned up properly. I've reverted this line back to the original.